### ISSUE TITLE
Upload ico logo form the admin portal 

### ISSUE THREAD

**Issue by vasa025** [Azure/Microsoft-commercial-marketplace-transactable-SaaS-offer-SDK#731]
As we have an option to upload the logo from the admin portal we need to upload the ico file from the admin portal.

**Comment by neelavarshad** [comment]
Yes, you can either upload from the admin site or change icons in the code repository and republish the web application.

**Comment by ottD** [comment]
@vasa025 You can already upload the ico file from the admin portal. Just select an .ico file instead of a .png. The admin portal code uploads it to the correct setting based on the file extension.


![Image](https://github.com/user-attachments/assets/c6f75be5-1227-484b-90a6-187ee4bdcb1a)


![Image](https://github.com/user-attachments/assets/2b6ccc86-fbca-49a6-8980-1d45abaa4b0f)
